includes/html/api_functions.inc.php

```diff
@@ -2319,6 +2319,12 @@ function list_logs(Illuminate\Http\Request $request, Router $router)
         $param[] = $to;
     }
 
+    $msg = $request->get('msg');
+    if ($msg) {
+        $query .= " AND `msg` LIKE '%" . $msg . "%'";
```

Comment:
The $param part below is correct, just use this for the query.
``` $query .= " AND `msg` LIKE '%?%'";```

This also needs a small documentation update so users know it exists.